deployments/rds-s3/terraform/rds-s3-components/main.tf

```diff
@@ -0,0 +1,329 @@
+  source            = "../../../../iaac/terraform/common/kubeflow-issuer"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/kubeflow-issuer"
+  }
+
+  addon_context = var.addon_context
+  depends_on = [kubernetes_namespace.kubeflow]
+}
+
+module "kubeflow_istio" {
+  source            = "../../../../iaac/terraform/common/istio"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/istio-1-14"
+  }
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_issuer]
+}
+
+module "kubeflow_dex" {
+  source            = "../../../../iaac/terraform/common/dex"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/dex"
+  }
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_istio]
+}
+
+module "kubeflow_oidc_authservice" {
+  source            = "../../../../iaac/terraform/common/oidc-authservice"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/oidc-authservice" 
+  }
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_dex]
+}
+
+module "kubeflow_knative_serving" {
+  source            = "../../../../iaac/terraform/common/knative-serving"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/knative-serving"
+  }  
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_oidc_authservice]
+}
+
+module "kubeflow_cluster_local_gateway" {
+  source            = "../../../../iaac/terraform/common/cluster-local-gateway"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/cluster-local-gateway"
+  }  
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_knative_serving]
+}
+
+module "kubeflow_knative_eventing" {
+  source            = "../../../../iaac/terraform/common/knative-eventing"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/knative-eventing"
+  }  
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_cluster_local_gateway]
+}
+
+module "kubeflow_roles" {
+  source            = "../../../../iaac/terraform/common/kubeflow-roles"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/kubeflow-roles"
+  }  
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_knative_eventing]
+}
+
+module "kubeflow_istio_resources" {
+  source            = "../../../../iaac/terraform/common/kubeflow-istio-resources"
+  helm_config = {
+    chart = "${var.kf_helm_repo_path}/charts/common/kubeflow-istio-resources"
+  }  
+  addon_context = var.addon_context
+  depends_on = [module.kubeflow_roles]
+}
+
+module "filter_kfp_set_values" {
+  source            = "../../../../iaac/terraform/utils/set-values-filter"
+  set_values = {
+    "rds.dbHost" = try(module.rds[0].rds_endpoint, null),
+    "s3.bucketName" = try(module.s3[0].s3_bucket_name ,null),
+    "s3.minioServiceRegion" = var.addon_context.aws_region_name,
+    "rds.mlmdDb" = var.mlmdb_name,
+    "s3.minioServiceHost" = var.minio_service_host
```

Comment:
aws_region_name? shouldnt this be S3 endpoint?
also expose minioServiceHost and minioServiceRegion to the top level for people using private link and non commercial regions

Reply:
> aws_region_name? shouldnt this be S3 endpoint?
s3.minioServiceRegion should be a region, s3.minioServiceHost is the endpoint

> also expose minioServiceHost and minioServiceRegion to the top level for people using private link and non commercial regions
done

Reply:
yep, you got my point